app.py

```diff
@@ -12,13 +11,16 @@ def index():
 
 @app.route('/video')
 def video():
+    print "Video request"
+
     FNULL = open(os.devnull, 'w')
 
-    hri_cmd = ['python', 'heart_rate_deamon.py', video_file, opencv_path]
-    opencv_ps = subprocess.Popen(hri_cmd, stdout=subprocess.PIPE, stderr=FNULL)
+    client_proc_cmd = 'python heart_rate_client.py ' + video_file  
```

Comment:
I thought of similar ideas (swapping out ZeroMQ for pure sockets / just getting the OpenCV to write it's output to the other process) but we really want to encapsulate it in one main process (as in app.py). 

There are 2 issues I can currently think of (and I can imagine a lot more) with letting the OpenCV server write or send stuff directly to ffmpeg. Firstly, we need information from the OpenCV server to start the ffmpeg process (width and height) so we either have to get the OpenCV server to send to 2 queues (app.py to start ffmpeg and raw data after to ffmpeg) or OpenCV could start ffmpeg. Both options introduce a new issue, how does OpenCV server know the ffmpeg process in the first case or how does the main server (app.py) know the ffmpeg in the second?

I think at the moment, I can't think of other, nicer ways of doing this than with subprocesses or threads.